### PR TITLE
S3 Service, interation with public bucket, controller to return word …

### PR DESCRIPTION
**Pull request by Astarch**
Added Library to use Amazon S3 in Java/Spring.

Created a Service which can crawl an S3 bucket, get its result files, iterate all entries and create a single map containing word->count data.

Created a Controller, available via `GET /api/s3/wordCountMap/amos-pretrendr-commoncrawl-sample`, which returns the result like this:
```
{
    "undermining": 9,
    "spilling": 7,
    "unavailable": 3,
    "abrupt": 3,
    "raions": 6,
    ....
}
```

### CODE REVIEW

File: backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3Service.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package de.pretrendr.businesslogic;
+
+import java.io.IOException;
+import java.util.Map;
+
+public interface S3Service {
+	public Map<String, Integer> getWorkdCountMapFromBucketName(String bucket_name) throws IOException;
```
Comment (pwillmann) [inline backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3Service.java]: this should probably be getWorkCount... right?

File: backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3ServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	public S3ServiceImpl() {
+		// AWSCredentials credentials = new BasicAWSCredentials("accessKey",
+		// "secretKey");
+		s3 = AmazonS3ClientBuilder.standard()// .withCredentials(new AWSStaticCredentialsProvider(credentials))
+				.withRegion(Regions.US_EAST_1).build();
+
+	}
+
+	@Override
+	public Map<String, Integer> getWorkdCountMapFromBucketName(String bucket_name) throws IOException {
```
Comment (pwillmann) [inline backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3ServiceImpl.java]: Same here, getWordCount I think

File: backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3Service.java (hunk 1 above)
Comment (Astarch) [reply backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3Service.java]: Typo fixed.

File: backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3ServiceImpl.java (hunk 2 above)
Comment (Astarch) [reply backendRestDraft/src/main/java/de/pretrendr/businesslogic/S3ServiceImpl.java]: Typo fixed.

**Pull request merged**